src/main/java/uk/gov/homeoffice/digital/sas/timecard/model/TimeEntry.java

```diff
@@ -48,10 +49,12 @@ public class TimeEntry extends BaseEntity {
     private String shiftType;
 
     @NotNull(message = "Actual start time should not be empty")
-    @Column(name = "actual_start_time")
+    @Column(name = "actual_start_time", columnDefinition = "TIMESTAMP WITH TIME ZONE")
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
     private Date actualStartTime;
 
-    @Column(name = "actual_end_time")
+    @Column(name = "actual_end_time", columnDefinition = "TIMESTAMP WITH TIME ZONE")
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
     private Date actualEndTime;
```

Comment:
I understand the Date type was introduced prior to this PR, but I do believe that `ZonedDateTime` type would be better aligned with the database type `TIMESTAMP WITH TIME ZONE`

Reply:
@daleelR , as discussed, this will be addressed in a separate change